bots/tests-invoke

```diff
@@ -182,9 +182,12 @@ class PullTask(object):
             # third-party project, link directly to text log
             status["link"] = "log"
         else:
-            # testing cockpit itself, use HTML log from current revision
+            # Testing cockpit itself, use HTML log from current
+            # revision.  Use log.html from code under test, but only
+            # if we are on master.  Other branches don't have a bots/
+            # in their repo.
             status["link"] = "log.html"
-            status["extras"] = [ "https://raw.githubusercontent.com/cockpit-project/cockpit/{0}/bots/task/log.html".format(self.github_revision) ]
+            status["extras"] = [ "https://raw.githubusercontent.com/cockpit-project/cockpit/{0}/bots/task/log.html".format(self.github_revision if not self.base else "master") ]
```

Comment:
This will go away with splitting out bots into a separate project. But I still wonder why we need that logic here -- why wouldn't this just be a hardcoded `master` instead of `{0}`?

Reply:
I think the idea originally was that you would be able to change log.html in a PR and immediately have it be used when testing the PR.

Reply:
I am ok with just hardcoding "master".

Reply:
Ah, I see, thanks! That's fine then. We'll need to revisit that when we split it out. In particular, log.html ought to move outside of bots/ into cockpit's tests/ somewhere. But until then this is a good enough fix.